api/adaptive_load/metrics_plugin_impl.proto

```diff
@@ -0,0 +1,13 @@
+// Plugin-specific config protos for MetricsPlugin.
+
+syntax = "proto3";
+
+package nighthawk.adaptive_load;
+
+// Example plugin-specific config proto, representing connection info for an
+// outside source of metrics. Part of an example showing how to create and
+// register a MetricsPlugin.
+message ExampleMetricsPluginConfig {
```

Comment:
What is our goal of submitting this example as part of our public API? Would it be better to just include these examples inside documentation, or at least in some non-public part of the codebase?

Reply:
Deleted this proto. But I kept the file with no protos so that we don't have to document how to create the file and add it to the build later.